modules/machines-edge-machine-pool.adoc

```diff
@@ -0,0 +1,103 @@
+
+:_content-type: CONCEPT
+[id="machines-edge-machine-pool_{context}"]
+= The edge machine pool
+
+{product-title} 4.12 introduced a new compute pool, _edge_, that is designed for use in remote zones. The edge compute pool configuration is common between AWS Local Zone locations. However, due to the image type and size limitations on Local Zone resources, the default instance type that is created can vary from the traditional worker pool.
+
+The default elastic block store (EBS) for Local Zone locations is `gp2`, which differs from the default worker pool. The preferred list of instance types for the edge compute pool follows the same order of worker pools. Depending on the availability of the location, the system chooses one of those instances. The preferred list of instance types for the edge compute pool includes:
+
+* `m6i.xlarge`
+* `m5.xlarge`
+* `c5d.2xlarge`
+
+The edge compute pool creates new labels that developers can use to deploy their applications onto those locations. The new labels are:
+
+* `node-role.kubernetes.io/edge=''`
+* `machine.openshift.io/zone-type=local-zone`
+* `machine.openshift.io/zone-group=<$ZONE_GROUP_NAME>`
+
+
+By default, the system creates the edge compute pool only if users add AWS Local Zone subnet IDs to the list `platform.aws.subnets`.
+
+The edge compute pool's machine sets have a `NoSchedule taint` by default to prevent regular workloads from being spread out on those machines. Users can only run user workloads if the tolerations are defined on the pod spec. This practice is demonstrated in the following examples:
```

Comment:
The examples below do not match this statement.

The examples are from `install-config.yaml`.

I think you can remove the last part of this phrase and create a new paragraph summarizing/introducing the examples below. wdyt?

Reply:
Works for me. Paragraph aside, do those examples all seem valuable to you? @mtulio

Reply:
if you would like to provide a full example, it's also fine for me. :)